src/psyclone/dynamo0p3_builtins.py

```diff
@@ -781,25 +810,25 @@ def gen_code(self, parent):
         :type parent: :py:class:`psyclone.f2pygen.BaseGen`
 
         '''
-        # In this case we're multiplying each element of a field by the
-        # supplied scalar value.
+        # In this case we're multiplying each element of a real-valued
+        # field by the supplied real scalar value.
         field_name = self.array_ref(self._arguments.args[1].proxy_name)
         scalar_name = self._arguments.args[0].name
         parent.add(AssignGen(parent, lhs=field_name,
                              rhs=scalar_name + "*" + field_name))
 
 
 # ------------------------------------------------------------------- #
-# ============== Dividing (scaled) fields =========================== #
+# ============== Dividing (scaled) real fields ====================== #
 # ------------------------------------------------------------------- #
 
 
 class DynXDividebyYKern(DynBuiltIn):
-    ''' Divide the first field by the second and return the result as
-    a third field '''
+    ''' Divide the first real-valued field by the second and return the
```

Comment:
Commas, full stop and blank line please.

Reply:
Done.